### DISCUSSION TITLE
Reload of tree

### DISCUSSION THREAD

**Discussion by blackhawk180** [mar10/wunderbaum#100] [Q&A]
I am used to work with Fancytree and i am now switching to wunderbaum.
In Fancytree it is possible to reload the tree via 
`
$.ui.fancytree.getTree("#tree").reload({
         url: sourceUrl;
});
`
Is there a way to do the same with the wunderbaum?

**Comment by mar10** [comment]
This should do the trick:
```js
mar10.Wunderbaum.getTree("#tree").load({ url: sourceUrl; });
```
